### ISSUE TITLE
Chapter 3 - Date range mismatch between prediction and test dataset 

### ISSUE THREAD

**Issue by mshearer0** [abdullahkarasan/mlfrm#4]
The prediction appears to be generated against the last n_steps of the training data (22/11 - 11/12) and then plotted against the date range of the test set (12/12 - 31/12)

These are the same length because the prediction is generated over n_steps = 13 which is same as the last 5% of the original 251 records selected for the test set.
